source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -23,7 +25,10 @@ This is the normal order we use when upgrading clusters, it does not have to exa
 
 ## Pods and HRs Health Checks
 
-Put a message out into the #cloud-native-announce channel to notify people which cluster will be upgraded (do not use @here as this is a notification not an announcement).
+Please post a notification in the #cloud-native-announce channel if you are undertaking any tasks related to the CFT clusters.
+Alternatively, utilize the #sds-cloud-native channel if you are involved in activities concerning SDS clusters. 
```

Comment:
Why the update? Would only include #sds-cloud-native if any addition.

Reply:
I have made the change again to explain why the sds channel should be used